freqtrade/migrate.py

```diff
@@ -0,0 +1,43 @@
+"""
+This script manages the migrations
+"""
+from pathlib import Path
+import os
+import caribou
+
+DB_NAME = 'trades'
+DB_PATH = '{}.sqlite'.format(DB_NAME)
+
+PARENT_DIR = os.path.abspath('.')
+MIGRATIONS_PATH = PARENT_DIR + '/migrations/'
+INITIAL_MIGRATION = '20171111135341'
+
+def run():
+    """
+    Run Migrations
+    """
+    # Check if old version of db is present
+    db_path_v2 = PARENT_DIR + '/{}v2.sqlite'.format(DB_NAME)
+    db_path_v3 = PARENT_DIR + '/{}v3.sqlite'.format(DB_NAME)
+
+    # If we use old db file format v2 or v3, rename it to new db filename
+    if Path(db_path_v2).is_file():
+        os.rename(db_path_v2, DB_PATH)
+        # Executes initial migration (catching up)
+        caribou.upgrade(DB_PATH, MIGRATIONS_PATH, INITIAL_MIGRATION)
+    elif Path(db_path_v3).is_file():
+        os.rename(db_path_v3, DB_PATH)
```

Comment:
I think this will break usage in Docker containers, where the database file is mounted into the container as stated in the [docs](https://github.com/gcarq/freqtrade/blob/develop/docs/installation.md#52-run-the-docker-image)  `-v ~/.freqtrade/tradesv3.sqlite:/freqtrade/tradesv3.sqlite`